browser/ui/BUILD.gn

```diff
@@ -764,10 +764,8 @@ source_set("ui") {
       "webui/playlist_ui.cc",
       "webui/playlist_ui.h",
     ]
-    deps += [
-      "//brave/components/playlist/mojom",
-      "//brave/components/playlist/resources:generated_resources",
-    ]
+    deps +=
+        [ "//brave/components/playlist/browser/resources:generated_resources" ]
```

Comment:
`playlist_ui.h/cc` includes playlist mojom and `playlist_service.h`, it means you also need `//brave/components/playlist/browser` here.

this `"ui"` target has `check_includes = false`, that's why gn check doesn't show anything here :/

Reply:
Than makes sense! Done.